### PR TITLE
Handle deleted/moved files from english documentation

### LINKED ISSUE #356: Mis translation in french
From manual page: https://php.net/function.random-int

---

but if I click on contribute to documentation, the translation of the text is correct : "Generates cryptographic random integers that are suitable for use where unbiased results are critical, such as when shuffling a deck of cards for a poker game. "

### PR DESCRIPTION
**Pull request by adoy**
Fixes #356

### CODE REVIEW

File: reference/random/constants.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<!-- Reviewed: no -->
+<appendix xml:id="random.constants" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ &reftitle.constants;
+ &extension.constants.core;
+ <para>
+  <variablelist>
+   <varlistentry xml:id="constant.mt-rand-mt19937">
+    <term>
+     <constant>MT_RAND_MT19937</constant>
+     (<type>entier</type>)
```
Comment (Girgias) [inline reference/random/constants.xml:12]:
```suggestion
     (<type>int</type>)
```

Please do not translate types within the ``<type>`` tags as those are used to generate links.

**Review by Girgias — CHANGES_REQUESTED** [review]
Other than the <type> issue LGTM from a quick review

File: reference/reflection/reflectionfunctionabstract/isstatic.xml [hunk 2]
```diff
@@ -26,10 +26,33 @@
...
+     <row>
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.1.0</entry>
+      <entry>
+       Cette méthode a été remontée. Auparavant, elle était uniquement définie dans
```
Comment (devnexen) [inline reference/reflection/reflectionfunctionabstract/isstatic.xml:47]: `remontée`. I guess you mean `retirée`

Comment (adoy) [reply reference/reflection/reflectionfunctionabstract/isstatic.xml:47]: No. It was previously on the `ReflectionMethod` and it was pulled up in the class hierarchy.

File: reference/random/constants.xml (hunk 1 above)
Comment (adoy) [reply reference/random/constants.xml]: Thanks @Girgias. I'm not sure `<type>` is used to generate link (at least not on [this page](https://www.php.net/manual/en/random.constants.php)) but nonetheless I'm ok with not translating those. I grepped the french doc and find some `<type>Tableau</type>`, `<type>chaîne de caractères</type>`, etc. I'll probably at some point do a commit to fix them.

File: reference/reflection/reflectionfunctionabstract/isstatic.xml (hunk 2 above)
Comment (devnexen) [reply reference/reflection/reflectionfunctionabstract/isstatic.xml:47]: My bad you re right

File: reference/random/constants.xml (hunk 1 above)
Comment (Girgias) [reply reference/random/constants.xml]: Yeah... known bug that at some point PhD broke and it stopped generating the links for int,float,bool,array, and string. But it does for ``<type>callable</type>``, ``<type>void</type>`` and others

**Pull request merged**